src/metrics/include/metrics/metrics.hpp

```diff
@@ -166,6 +166,7 @@ class RegistryT final {
                        bool assertMetricsBuffer = true)
         : _format{std::move(format)}, _pathPrefix{std::move(pathPrefix)} {
         if (_format.useGrpc()) {
+            boost::filesystem::remove_all(_pathPrefix);
```

Comment:
Do we need this now that we've set the format to csv for test actors?

Alternatively maybe we can have a `CLEAR_POPLAR_METRICS` env var that this checks or something? I think most users will not want to delete prior metrics data. My WIP change jumps through some hoops to look for existing data and moves it to a timestamp'ed dir (does this on the genny python side) so it's really only tests that will need this remove functionality.

Reply:
Yeah it's not necessary; I had left it in since there's a relevant ticket but I like your approach. Removed.